### PR TITLE
deconflict icon added to position manager bidder lists

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/472)

### CODE REVIEW

File: src/Constants/Tooltips.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
```
Comment (elizabeth-jimenez) [inline src/Constants/Tooltips.js:1]: 🤷

Comment (mjoyce91) [reply src/Constants/Tooltips.js:1]: I don't really like this rule, but basically it wants you to make `DECONFLICT_TOOLTIP_TEXT` your default export

```js
const DECONFLICT_TOOLTIP_TEXT = {
  title: 'Deconflict Alert',
  text: 'Candidate Ranked #1 on, at least, one other Short List',
};

export default { DECONFLICT_TOOLTIP_TEXT };
```

Comment (mjoyce91) [reply src/Constants/Tooltips.js:1]: Up to you, I'm fine with the eslint-disable

Comment (elizabeth-jimenez) [reply src/Constants/Tooltips.js:1]: oh nice, i'll do it the way you have above 🥳  thanks!

**Review by scott062 — APPROVED** [review]
👍

**Pull request merged**